.github/workflows/comment.yaml

```
@@ -14,6 +14,7 @@ on:
       - "Express GraphQL Test"
       - "Federation JVM Test"
       - "GQLGen Test"
+      - "Grafbase test"
```

I believe this is case sensitive so probably thats why comments were not triggered
```suggestion
      - "Grafbase Test"
```